packages/office-addin-sso/package-lock.json

```diff
@@ -316,24 +323,16 @@
 			}
 		},
 		"node_modules/acorn": {
-			"version": "4.0.13",
-			"resolved": "https://registry.npmjs.org/acorn/-/acorn-4.0.13.tgz",
-			"integrity": "sha1-EFSVrlNh1pe9GVyCUZLhrX8lN4c=",
+			"version": "7.4.1",
+			"resolved": "https://registry.npmjs.org/acorn/-/acorn-7.4.1.tgz",
+			"integrity": "sha512-nQyp0o1/mNdbTO1PO6kHkwSrmgZ0MT/jCCpNiwbUjGoRN4dlBhqJtoQuCnEOKzgTVwg0ZWiCoQy6SxMebQVh8A==",
 			"bin": {
 				"acorn": "bin/acorn"
 			},
 			"engines": {
 				"node": ">=0.4.0"
 			}
 		},
-		"node_modules/acorn-globals": {
-			"version": "3.1.0",
-			"resolved": "https://office.pkgs.visualstudio.com/_packaging/Office/npm/registry/acorn-globals/-/acorn-globals-3.1.0.tgz",
```

Comment:
This was the problem where it was pulling from the Office npm registry, which it shouldn't.